tests/unit/hive_metastore/test_table_migrate.py

```diff
@@ -32,20 +32,38 @@ def test_migrate_managed_tables_should_produce_proper_queries():
     table_mapping = create_autospec(TableMapping)
     table_mapping.get_tables_to_migrate.return_value = [
         TableToMigrate(
-            Table("hive_metastore", "db1_src", "managed_src", "MANAGED", "DELTA"),
-            Rule("workspace", "ucx_default", "db1_src", "db1_dst", "managed_src", "managed_dst"),
-        )
+            Table("hive_metastore", "db1_src", "managed_dbfs", "MANAGED", "DELTA", "dbfs:/some_location"),
+            Rule("workspace", "ucx_default", "db1_src", "db1_dst", "managed_dbfs", "managed_dbfs"),
+        ),
+        TableToMigrate(
+            Table("hive_metastore", "db1_src", "managed_mnt", "MANAGED", "DELTA", "s3:/mnt/location"),
+            Rule("workspace", "ucx_default", "db1_src", "db1_dst", "managed_mnt", "managed_mnt"),
+        ),
+        TableToMigrate(
+            Table("hive_metastore", "db1_src", "managed_other", "MANAGED", "DELTA", "s3:/location"),
+            Rule("workspace", "ucx_default", "db1_src", "db1_dst", "managed_other", "managed_other"),
+        ),
     ]
     table_migrate = TablesMigrate(table_crawler, client, backend, table_mapping)
     table_migrate.migrate_tables()
 
-    assert (list(backend.queries)) == [
```

Comment:
Why do we need to split this statement? It's more readable this way

Reply:
I assume you meant to break off the assert, in which case I have.